libs/src/api/user.js

```diff
@@ -1,7 +1,24 @@
+const _ = require('lodash')
 const { Base, Services } = require('./base')
 const Utils = require('../utils')
 const CreatorNode = require('../services/creatorNode')
 
+const USER_PROPS = [
+  'is_creator',
+  'is_verified',
+  'name',
+  'handle',
+  'profile_picture_sizes',
+  'cover_photo_sizes',
+  'bio',
+  'location',
+  'creator_node_endpoint'
+]
+const USER_REQUIRED_PROPS = [
+  'name',
+  'handle'
+]
```

Comment:
same thing here. creatornode lists the user props as
```
user_metadata_format = {
    "profile_picture": None,
    "profile_picture_sizes": None,
    "cover_photo": None,
    "cover_photo_sizes": None,
    "bio": None,
    "name": None,
    "location": None,
    "handle": None
}
```
don't think we need things like creator_node_endpoint or is_creator in metadata since it's on chain
and it has no required props. if we require name that's fine but handle is onchain anyway so i think we can safely remove this from required

Reply:
Also, didn't want to change the metadata here from what it currently is. I think we should rework the schema validation and what fields are sent where, but I'm just copying what's already in this file

Reply:
Added back profile_picture and cover_photo